### PR TITLE
Added get_arm_info

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: sdk/core/azure-mgmt-core/azure/mgmt/core/tools.py [hunk 1]
```diff
@@ -217,3 +218,29 @@ def is_valid_resource_name(rname: str, exception_type: Optional[Type[BaseExcepti
     if exception_type:
         raise exception_type()
     return False
+
+
+def get_arm_info(cloud_setting: AzureClouds) -> Dict[str, Any]:
```
Comment (kashifkhan) [inline sdk/core/azure-mgmt-core/azure/mgmt/core/tools.py:223]:
```suggestion
def get_arm_info(cloud_setting: AzureClouds) -> Dict[str, str]:
```

to match the docstring

Comment (kashifkhan) [reply sdk/core/azure-mgmt-core/azure/mgmt/core/tools.py:223]: actually maybe the docstring is wrong here as the return also has a list value

Comment (xiangyan99) [reply sdk/core/azure-mgmt-core/azure/mgmt/core/tools.py:223]: Yes. I updated the return type but forgot to update the docstring.

Comment (kashifkhan) [inline sdk/core/azure-mgmt-core/azure/mgmt/core/tools.py:223]: lets rename this to `get_arm_endpoints` - it adds clarity as to what youll get back,

**Review by kashifkhan — APPROVED** [review]

**Pull request merged**